client/src/components/graph/graph.tsx

```diff
@@ -727,33 +797,37 @@ class Graph extends React.Component<GraphProps, GraphState> {
     }
   }
 
-  mapScreenToPoint(pin: [number, number]) {
+  mapScreenToPoint(pin: [number, number]): vec2 {
     /*
         Map an XY coordinates from screen domain to cell/point range,
         accounting for current pan/zoom camera.
         */
     const { camera, projectionTF, modelInvTF, viewport } = this.state;
-    const cameraInvTF = camera!.invView();
+    const cameraInvTF = camera ? camera.invView() : null;
     /* screen -> gl */
     const x = (2 * pin[0]) / viewport.width - 1;
     const y = 2 * (1 - pin[1] / viewport.height) - 1;
     const xy = vec2.fromValues(x, y);
     const projectionInvTF = mat3.invert(mat3.create(), projectionTF);
     vec2.transformMat3(xy, xy, projectionInvTF);
-    vec2.transformMat3(xy, xy, cameraInvTF);
+    if (cameraInvTF) {
+      vec2.transformMat3(xy, xy, cameraInvTF);
+    }
     vec2.transformMat3(xy, xy, modelInvTF);
     return xy;
   }
 
-  mapPointToScreen(xyCell: [number, number]) {
+  mapPointToScreen(xyCell: [number, number]): [number, number] {
     /*
         Map an XY coordinate from cell/point domain to screen range.  Inverse
         of mapScreenToPoint()
         */
     const { camera, projectionTF, modelTF, viewport } = this.state;
-    const cameraTF = camera!.view();
+    const cameraTF = camera?.view() || undefined;
```

Comment:
since we have optional chaining already, we don't have to do `|| defined`, since TS transpiles `camera?.view()` to `camera ? camera.view() : camera` in JS automatically